recipe-server/normandy/recipes/api/serializers.py

```diff
@@ -170,6 +189,9 @@ class Meta:
             'date_created',
             'recipe',
             'comment',
+            'is_approved',
+            'is_rejected',
+            'is_pending_approval',
```

Comment:
Instead of 3 booleans, what about something like `approval`, that has multiple states, `approved`, `rejected`, or `pending`?